### ISSUE TITLE
`ds.concat`doesn't accept the `name` argument | BigQuery

### ISSUE THREAD

**Issue by amirbtb** [rasgointelligence/RasgoQL#60]
Hello,

I tried to use the `ds.conct()` method as shown in the [Example of the documentation](https://docs.rasgoql.com/transforms/column-transforms/concat#example).
`ds.concat(concat_list=['first_column',"'-'",'second_column'], name="both_columns")` returns the following error : 
```
[File /usr/local/lib/python3.8/dist-packages/rasgoql/primitives/transforms.py:54, in TransformableClass._create_aliased_function.<locals>.f(*arg, **kwargs)
     ]()[53](file:///usr/local/lib/python3.8/dist-packages/rasgoql/primitives/transforms.py?line=52)[ def f(*arg, **kwargs) -> 'SQLChain':
---> ]()[54](file:///usr/local/lib/python3.8/dist-packages/rasgoql/primitives/transforms.py?line=53)[     return self.transform(name=transform.name, *arg, **kwargs)

TypeError: transform() got multiple values for keyword argument 'name']()
```
When I don't provide the `name` argument, the functions works.

Thanks again !

**Comment by griffatrasgo** [comment]
Hi @amirbtb you've been busy testing! Thank you for using the package.

This seems to be the most straightforward of the 3 issues, so let's start with this one. The root cause is we're using a transform arg that conflicts with one of the transform function parameters. The fastest fix will be to rename the `name` arg in `concat` to something break-y.

We can get a patch out for this one in under a day. I'll ping you here when it's ready.

**Comment by griffatrasgo** [comment]
Hi @amirbtb we pushed a fix for this in rasgotransforms version 1.1.2. The `name` argument on the `concat` transform is now `alias`.

If you run `pip install rasgotransforms --upgrade` this code should work for you:
```
ds.concat(concat_list=['first_column', "'-'" ,'second_column'], alias="both_columns")
```

**Comment by amirbtb** [comment]
Hi @griffatrasgo it works fine with the `rasgotransforms==1.1.2` & the `alias` argument. Thank you for this fix !
